### PR TITLE
seq2seq by yeonsik

### PR DESCRIPTION
**Pull request by justHungryMan**

### CODE REVIEW

File: trainer/model/seq2seq.py [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+
+        #encoder의 hidden_dim과 decoder의 hidden_dim은 같아야함.(n_layers도)
+        assert encoder.hidden_dim == decoder.hidden_dim
+        assert encoder.n_layers == decoder.n_layers
+
+    def forward(self, src, trg, teacher_forcing_ratio=0.5):
+        #src = (src len, batch_size)
+        #trg = (trg len, batch_size)
+        trg_len = trg.shape[0]
+        batch_size = trg.shape[1]
```
Comment (justHungryMan) [inline trainer/model/seq2seq.py]: trg_len -> trg.shape[1]
batch_size -> trg.shape[0]
확인

File: trainer/model/seq2seq.py [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+        trg_vocab_size = self.decoder.ouput_dim
```
Comment (justHungryMan) [inline trainer/model/seq2seq.py:55]: vocab_size 가 무엇?

File: trainer/model/seq2seq.py [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+        #Encoder의 output_dim만큼의 크기를 받아, embedding_dim만큼의 크기로 Embedding.
+        self.embedding = nn.Embedding(output_dim, embedding_dim)
+        #embedding_dim 크기를 받아, hidden_dim 크기의 은닉층 생성, rnn 층은 n_layers만큼.
+        self.rnn = nn.LSTM(embedding_dim, hidden_dim, n_layers, dropout=dropout)
+        #hidden_dim을 입력받아 output_dim을 출력
+        self.fc_out = nn.Linear(hidden_dim, output_dim)
+        self.dropout = nn.Dropout(dropout)
+
+    def forward(self, input, hidden, cell):
+        input = input.unsqueeze(0)
```
Comment (justHungryMan) [inline trainer/model/seq2seq.py:33]: unsqueeze?

File: trainer/model/seq2seq.py [hunk 4]
```diff
@@ -0,0 +1,90 @@
...
+        super().__init__()
+
+        self.output_dim = output_dim
+        self.hidden_dim = hidden_dim
+        self.n_layers = n_layers
+
+        #Encoder의 output_dim만큼의 크기를 받아, embedding_dim만큼의 크기로 Embedding.
+        self.embedding = nn.Embedding(output_dim, embedding_dim)
+        #embedding_dim 크기를 받아, hidden_dim 크기의 은닉층 생성, rnn 층은 n_layers만큼.
+        self.rnn = nn.LSTM(embedding_dim, hidden_dim, n_layers, dropout=dropout)
```
Comment (justHungryMan) [inline trainer/model/seq2seq.py]: dropout initialize
